pdns/dnssecinfra.cc

```diff
@@ -291,6 +316,103 @@ bool DNSCryptoKeyEngine::testOne(int algo)
   return ret;
 }
 
+static map<string, string> ICSStringToMap(const string& argStr)
```

Comment:
`ICS` should be `ISC`.

I know you did not introduce this duplication, but it looks like `DNSCryptoKeyEngine::makeFromISCString` might be reusable here. Not necessary for this PR.